potpie/context-engine/src/potpie_context_engine/adapters/outbound/connectors/gitlab/api_client.py

```
@@ -0,0 +1,787 @@
+``author``, ``labels``, …) so playbooks and graph mutations share one
+vocabulary across forges. GitLab-native fields that have no GitHub
+equivalent (approvals, resolvable discussion threads, weight, time
+tracking) are added on top rather than dropped.
+
+Verified against GitLab CE 19.2 (REST API v4).
+"""
+
+from __future__ import annotations
+
+import logging
+from typing import Any, Iterator, Protocol
```

Ruff (0.16.1)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)